packages/devtools/src/launcher.ts

```diff
@@ -89,7 +94,7 @@ async function launchChrome (capabilities: ExtendedCapabilities) {
             '--headless',
             '--no-sandbox'
         ] : []),
-        ...(chromeOptions.args || [])
+        ...(chromeOptionsArgs || [])
```

Comment:
Instead of the 3 lines above we can just do:

```suggestion
        ...(chromeOptions.args || []).map((a) => a.startsWith('--') ? a : `--${a}`)
```